.github/workflows/testground.yml

```diff
@@ -0,0 +1,53 @@
+name: Run Testground
+
+on:
+  push:
+    branches: ['**'] # TODO: Restrict this to main
+jobs:
+  run-testground:
+      runs-on: ubuntu-latest
+      steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: "^1.19.0"
+      # Get testground and our test plan code
+      - uses: actions/checkout@v3
+        with:
+         repository: 'statechannels/testground'
+         path: "code/testground"
+      - uses: actions/checkout@v3
+        with:
+          repository: 'statechannels/go-nitro-testground'
+          path: "code/go-nitro-testground"
+          ref: update-to-main #TODO Remove when https://github.com/statechannels/go-nitro-testground/pull/103 is merged
```

Comment:
This is required because `main` on `go-nitro-testground` hasn't been updated to work with the latest `go-nitro` (see https://github.com/statechannels/go-nitro-testground/pull/103)